blocks/editable/index.js

```diff
@@ -430,12 +516,26 @@ export default class Editable extends Component {
 		}
 	}
 
+	/**
+	 * Handles tinyMCE key up event
+	 *
+	 * @param {number} keyCode The key code that has been pressed on the keyboard.
+	 */
 	onKeyUp( { keyCode } ) {
 		if ( keyCode === BACKSPACE ) {
 			this.onSelectionChange();
 		}
 	}
 
+	/**
+	 * @summary Splits the content at the location of the selection.
```

Comment:
As a note of consistency, we've not tended to use the `@summary` keyword. I'd be curious to know if most parsers could infer the summary by newline separation, or if we could apply this ourselves in however we intend to use the descriptions.

It also falls a bit out of place with the other tags being below description; which is more sensible in this case since it's likely the first thing a developer would want to read.

http://usejsdoc.org/tags-summary.html

Reply:
I've removed all `@summary` statements.